output/cloud/expv2/output.go

```diff
@@ -120,7 +117,7 @@ func (o *Output) AddMetricSamples(s []metrics.SampleContainer) {
 	// evaluate to do something smarter, maybe having a lock-free
 	// queue.
 	select {
-	case <-o.stopSendingMetrics:
+	case <-o.stopMetricsCollection:
```

Comment:
Wasn't this somewhat misnamed in the first place? From the context, I tend to think that it should be named `stopSamplesCollection`, or `stopSamplesAggregation` instead. What do you think? Am I missing something?

Reply:
Moved this to https://github.com/grafana/k6/pull/3082, let's discuss there